assets/javascripts/components/LinkCounter/index.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+export const DEFAULT_SELECTOR = 'div.link-counter';
+
+const DEFAULT_OPTIONS = {
+    toggleActiveAt: '.link-counter__title',
+    activeClass: 'link-counter__item--active'
+}
+
+class LinkCounter {
```

Comment:
This component could have been made more generic e.g. collapsible so that it can be re-used. It's compulsory to change but this is a general best practice we should follow when implementing UI components.

Reply:
@olivierobert I've changed from `LinkCount` to `Collapsible`  already 🙌